GetStream/stream-chat-android
# Fix/compose user filters

### 🎯 Goal

Fixes an issue where, when we log in, the user is still not present in the the Client/State and this makes it so that we return an empty user ID `in` filter for `members` - no channels are returned, even if the user is logged in after a while.

### 🧪 Testing

1. Run compose sample.
2. Log in with a user.
3. Channels should load.

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
